README.md

```diff
@@ -66,6 +67,13 @@ Then run `grunt monitor` to compile the documentation. When it finishes, open a
 ### Documentation for previous releases
 Previous releases and their documentation are available for [download](https://github.com/materializecss/materialize/releases).
 
+## Build
+If you want to get `materialize.css` or `materialize.js` which generated from the latest commit, you can build with the following command after `npm install`. See `package.json` to check the current version like `1.0.0`.
+
+```sh
+npx grunt release --oldver=<current_version> --newver=<new_version>
```

Comment:
hmm pardon me, but I don't get it why we need both `oldver` and `newver` arguments? 😕

Reply:
It could be omited, but in my opinion, it would be good for the future contributors to show the command of the actual version upgrade work. In addition, it will be written as "vundefined" in the header when omit `newver`, it looks not nice.

Reply:
We should probably add this as command to the package.json file.

Reply:
@DanielRuf 
OK, I added build script in `package.json`, and use npm instead of npx for consistency.

Reply:
hmm I got what `newver` arg means, but how about `oldver`? Doesn't seem to affect anything? 😕